rclrs/src/parameter/value.rs

```diff
@@ -392,6 +498,35 @@ impl ParameterValue {
             unreachable!()
         }
     }
+
+    pub(crate) fn rcl_parameter_type(&self) -> u8 {
+        match self {
+            ParameterValue::Bool(_) => ParameterType::PARAMETER_BOOL,
+            ParameterValue::Integer(_) => ParameterType::PARAMETER_INTEGER,
+            ParameterValue::Double(_) => ParameterType::PARAMETER_DOUBLE,
+            ParameterValue::String(_) => ParameterType::PARAMETER_STRING,
+            ParameterValue::ByteArray(_) => ParameterType::PARAMETER_BYTE_ARRAY,
+            ParameterValue::BoolArray(_) => ParameterType::PARAMETER_BOOL_ARRAY,
+            ParameterValue::IntegerArray(_) => ParameterType::PARAMETER_INTEGER_ARRAY,
+            ParameterValue::DoubleArray(_) => ParameterType::PARAMETER_DOUBLE_ARRAY,
+            ParameterValue::StringArray(_) => ParameterType::PARAMETER_STRING_ARRAY,
+        }
+    }
+
+    /// Returns the `ParameterKind` for the parameter as if it was a static parameter.
+    pub(crate) fn static_kind(&self) -> ParameterKind {
```

Comment:
The name `static_kind` feels a bit odd to me. I would suggest renaming it to the `current_kind` or even just `kind` because this is a method on the `ParameterValue` enum, and there's no such thing as a "dynamic" value (the word "dynamic" only makes sense when talking about a parameter, not a specific value).

Reply:
https://github.com/ros2-rust/ros2_rust/pull/342/commits/a7d1befbe078418fca4852b200e3e1b6c4a62f9f